objects/AllPlayerCards.15bb07/ShortSupply.e5f541.ttslua

```diff
@@ -0,0 +1,57 @@
+local MAT_GUIDS = { "8b081b", "bd0ff4", "383d8b", "0840d5" }
```

Comment:
Should update this to use the Playermat API

Reply:
I was unclear, sorry :)  I didn't mean just this line, I meant most of the operations here.  This gets the mat ref through the API, but after that it's still doing the operations via call() and getVar on the mat itself.

getMatByPosition() should be a function in the API, as it's likely to see general use (e.g. customized cards will need it).  The actual operation here (discarding X from the top of the deck) could be generalized through the API as well, maybe something like discardFromDeck(count, completePartial) - second parameter there specifying whether the operation should continue if the deck is < count.

A good rule of thumb is that anywhere we see call() or setVar() or getVar() etc we should seriously look at moving those operations to the Playermat.

Reply:
That makes sense. I'll update this PR when I get to it.